ci/get_labels/main.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+var prUrl = flag.String("url", "", "Url of pull request")
+
+func main() {
+	flag.Parse()
+	labelsUrl := strings.Replace(*prUrl, "/pulls/", "/issues/", 1)
+	labelsUrl += "/labels"
+	resp, err := http.Get(labelsUrl)
+	if err != nil {
+		panic(err)
+	}
+	defer resp.Body.Close()
+
+	type Labels struct {
```

Comment:
Would prefer this type to be defined at the top or for this to be an anonymous struct in the declaration on line 41. Probably also doesn't need to be capitalized